service/awsclusterconfig/v1/version_bundle.go

```diff
@@ -0,0 +1,20 @@
+package v1
+
+import (
+	"time"
+
+	"github.com/giantswarm/versionbundle"
+)
+
+func VersionBundle() versionbundle.Bundle {
+	return versionbundle.Bundle{
+		Changelogs:   []versionbundle.Changelog{},
+		Components:   []versionbundle.Component{},
+		Dependencies: []versionbundle.Dependency{},
+		Deprecated:   false,
+		Name:         "cluster-operator",
+		Time:         time.Date(2018, time.March, 27, 12, 00, 0, 0, time.UTC),
+		Version:      "0.1.0",
```

Comment:
Should versioning start from 1.0.0 right away or is 0.1.0 ok for now?

Reply:
Lets start with `0.1.0`.